Updates to documentation for initial setup

aglyle opened the pull request:
Adds documentation to bypass setup errors.

e-t-k commented on treeshop.md:
```diff
@@ -26,15 +26,20 @@ Clone this repository:
     cp samples/*.fastq.gz treeshop/primary/original/TEST/
 
-Spin up a single cluster machine:
+Spin up a single cluster machine (make sure you have created your SSH key):
```
Provide instructions for:
a) checking if you already have an "~/.ssh/id_rsa" private+public keypair files that don't require a password
b) if you don't have a keypair, how to create it

e-t-k replied:
Another thing to incorporate at the end: 
- how to spin down machines when done
 - the info from Rob's April 12 email "Floating IPs" about how to clear the IP addresses after spinning down machines.

e-t-k commented:
Thank you @aglyle ; I'm satisfied.

@klearned , passing this to you -- could you please do a readthrough of the new treeshop.md for potential improvements?

klearned commented:
Yes, I'll read through the new treeshop.md today

aglyle commented:
Thanks for the comments @klearned!
@e-t-k can you take a final look at these changes and I'll ping Rob to merge the branch.

e-t-k commented:
@aglyle looks great. I approve this doc.

aglyle commented:
@rcurrie Updates to the treeshop.md ready to be merged with master.  Let me know if there's anything you want me to change.

rcurrie commented:
Good work - thank you for filling in pieces. I made a bunch of comments (feel free to push back!). In general I want to be careful about helping someone use this vs. helping someone that shouldn’t be using this.

> On May 31, 2018, at 12:35 PM, Geoff Lyle <notifications@github.com> wrote:
> 
> @rcurrie <https://github.com/rcurrie> Updates to the treeshop.md ready to be merged with master. Let me know if there's anything you want me to change.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/UCSC-Treehouse/pipelines/pull/13#issuecomment-393652268>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAeEGjVKXYBzwy1aThYuhOxVnJIVkIVYks5t4EXngaJpZM4TW7PA>.
>

aglyle commented:
@rcurrie Incorporated your comments.  Removed sections that we expect users to know (SSH Key creation), unsure if you wanted to keep PATH setup or not, but it's removed in the latest version.  Also moved documentation for shutting down select machines and different fabfiles to links in the options.  Removed shutting down IPs as it is Treehouse specific.
